orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/AuthenticationAware.kt

```diff
@@ -39,6 +40,16 @@ interface AuthenticationAware {
       allowedAccounts = getExecution().getAuthentication()?.allowedAccounts
     }
 
-    AuthenticatedRequest.propagate(block, false, currentUser).call()
+    try {
+      ExecutionContext.set(ExecutionContext(
+        getExecution().getApplication(),
+        getExecution().javaClass.simpleName.toLowerCase(),
+        getExecution().getId(),
```

Comment:
actually going to remove stageId instead ... agree with the sentiment that we might have problems if we flop between replicas between stages.